### DISCUSSION TITLE
!commands

### DISCUSSION THREAD

**Discussion by starryskies2** [runelite/runelite#15379] [Ideas]
was wondering if there is a way to add to the commands waves at the inferno/fight caves? and if there is can I help somehow?

**Comment by Nightfirecat** [comment]
Do you mean a `!waves` command? I don't think we're interested in making that. You could use a hub plugin such as the [Fight Cave Waves plugin](https://runelite.net/plugin-hub/show/fight-cave-waves) to show the current wave in an overlay.

**Reply by starryskies2** [reply]
i meant for example typing command like ```!wave inferno``` and it will post which wave are you at in the inferno

**Reply by Casesos** [reply]
The time to type //!wave Inferno, you could just type your wave

**Reply by Nightfirecat** [reply]
I don't think we're interested in creating something which just broadcasts your wave. You can always look in your chatbox to see which wave you're on and type that in for others to see.

**Comment by Drklich256** [comment]
Make a code for chrome os smh I'm
 fucking pissed
